plugins/ipc/ipc-helpers.hpp

```diff
@@ -55,7 +55,7 @@ WFJSON_GETTER_FUNCTION(bool, bool);
 #undef WFJSON_GETTER_FUNCTION
 
 
-inline wayfire_view find_view_by_id(uint32_t id)
+inline wayfire_view find_view_by_id(const std::optional<uint64_t>& id)
```

Comment:
this makes no sense at all as a change, the validation of id should be done before looking for the view, so that the user can differentiate. I.e if the user doesn't specify neither `id` nor `view-id`, then the error should be `missing id field` not `view not found` bc the ipc client did not even specify which client